functions/index.js

```diff
@@ -1,53 +1,78 @@
-const functions = require("firebase-functions");
-const admin = require("firebase-admin");
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
 
 admin.initializeApp();
 
 const db = admin.firestore();
 
-exports.onSignup = functions.auth.user().onCreate(async (user) => {
-    async function generateUniqueInviteCode() {
-        const length = 8;
-        const dictionary = "234579abcdefghjkmnpqrstwxyz"; // unambiguous characters
+exports.onSignup = functions.auth
+	.user()
+	.onCreate(async user => {
+		async function generateUniqueInviteCode() {
+			const length = 8;
+			const dictionary =
+				'234579abcdefghjkmnpqrstwxyz'; // unambiguous characters
 
-        const generateRandom = () => {
-            let out = "";
-            for (let i = 0; i < length; i++) {
-                out += dictionary.charAt(Math.floor(Math.random() * dictionary.length));
-            }
-            return out;
-        };
-        const checkIsRandom = (allegedRandom) => { // dumbest way of guaranteeing uniqueness but this will have to do for now
-            return new Promise((resolve, reject) => {
-                db.collection("users").where("inviteCode", "==", allegedRandom)
-                    .get()
-                    .then(querySnapshot => resolve(querySnapshot.empty))
-                    .catch(reject);
-            });
-        };
+			const generateRandom = () => {
```

Comment:
Could we actually use space rather than tab here for indentation? This looks a bit too much space for each new line.